README.md

```diff
@@ -63,7 +63,8 @@ Focus Area | Goal
 --- | ---
 [Code Development Activity](./code_development_activity.md) | Learn about the types and frequency of activities involved in developing code.
 [Code Development Efficiency](./code_development_efficiency.md) | Learn how efficiently activities around code development get resolved.
-[Code Development Process Quality](./code_development_quality.md) | Learning about the processes to improve/review quality that are used (for example: testing, code review, tagging issues, tagging a release, time to response, CII Badging).
+[Code Development Process Quality](./code_development_process_quality.md) | Learn about the processes to improve/review quality that are used (for example: testing, code review, tagging issues, tagging a release, time to response, CII Badging).
+[Issue Resolution](./issue_resolution.md) | Identify how effective the community is at addressing issues identified by community participants.
```

Comment:
Please double-check the links in the ./README.md 
I think they are missing the directory where the file is located.